library_generation/utilities.sh

```diff
@@ -56,23 +56,18 @@ unzip_src_files() {
 
 # get gapic options from .yaml and .json files from proto_path.
 get_gapic_opts() {
-  local gapic_config
-  local grpc_service_config
-  local api_service_config
-  gapic_config=$(find "${proto_path}" -type f -name "*gapic.yaml")
-  if [ -z "${gapic_config}" ]; then
-    gapic_config=""
-  else
-    gapic_config="gapic-config=${gapic_config},"
-  fi
-  grpc_service_config=$(find "${proto_path}" -type f -name "*service_config.json")
```

Comment:
Can we keep these logics that try to find the file in current folder if not specified? It solves 95% of the case, and can reduce the Bazel configs in the future.

Reply:
done.